profile.ps1

```diff
@@ -1055,6 +1059,61 @@ function Start-ElevatedConsole {
     Start-Process (Get-Process -Id $PID).Path -Verb RunAs -ArgumentList @("-NoExit", "-Command", "Set-Location '$($PWD.Path)'")
 }
 
+function Export-Branch {
+    [OutputType([void])]
+    [Alias("git-fire")]
+    param(
+        [string] $Message,
+
+        [int] $ShutdownDelay = 15
+    )
+
+    begin {
+        $Author = $(git config user.name)
+        $Remotes = $(git remote)
+        $CurrentBranch = $(git branch --show-current)
+        $NewBranch = "fire/$CurrentBranch/${env:COMPUTERNAME}"
```

Comment:
This branch is not guaranteed to be unique if multiple machines were to have the same client name. Appending `/$env:USERNAME` or `/$Author` in the branch name  would be one possible solution, but not without sanitizing the branch name first because there are a couple of characters that are prohibited, such as
- `..`
- `:`
- `~`
- `^`
- `?`
- `*`
- `[`
- `//`
- ` ` (space character)
- ASCII Control Characters such as
- cannot start with a `dash`

Note that this list is not exhaustive. But we can use the command

```powershell
$IsValidBranchName = $(git check-ref-format --branch $NewBranch 2&>1) -eq $NewBranch
```

to check whether a branch name is valid or not.

Source: https://stackoverflow.com/a/3651867/10827244